draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -520,10 +520,7 @@ necessary frames have been received on stream zero MUST also result in a stream
 error of type `PROTOCOL_ERROR`.
 
 The referenced certificate chain MUST conform to the requirements expressed in
-the `CERTIFICATE_REQUEST` to the best of the sender's ability. Specifically, if
-the `CERTIFICATE_REQUEST` contained a non-empty `Cert-Extensions` element, the
-end-entity certificate MUST match with regard to the extensions recognized by
-the sender.
+the `CERTIFICATE_REQUEST` to the best of the sender's ability.
```

Comment:
This is really a requirement on exported authenticators.  Can we turn this into a statement of truth instead?  Or remove it entirely.

Reply:
I'm not sure that's true -- the exported authenticator functionality is responsible for communicating the requirements and the certificate, but it's not responsible for selecting the certificate to respond with.  I think it's a requirement on whomever does certificate selection, which I believe is the HTTP layer.

Reply:
It's not really actionable though.  If your intent is to state an interoperability requirement (why else do you use MUST), what do we lose if someone violates that?  I mean, I know that NSS could try harder to comply, but we do have better things to do than add full support for oid_filter (just for example, which is really hard to implement in practice).